### PR TITLE
Re-factor: Reacttabular styles and table actions

### PR DESCRIPTION
**Pull request by caleybrock**
This PR does:
- create a common set of styles for components using react-tabular
- make `ManageStudentsTable`, `PersonalProjectsTable`, and `SectionTable` use the common styles
- moves `QuickActions` to a common table location so other tables that need actions can use it (todo: add to `ManageStudentsTable`)

This PR does NOT:
- touch `SectionsTable` (not to be confused with `SectionTable`), which does not use react-tabular, but still has a set of table styles

What's next?
- [owner: me] continue to make `QuickActions` more re-usable by also creating a common child component
- [owner: unknown] convert `SectionsTable` to use react-tabular/tableConstants to keep it consistent with the rest of our tables
- [owner: poorva to decide if we should] update `ProjectList` with the new styles, since it already uses react-tabular, but has angular styles. Mark said to hold off on this one until getting specific instruction, but this would be a very simple work item now

Screenshots:
ManageStudentsTable
screen shot 2017-10-19 at 3 57 07 pm

PersonalProjectsTable
screen shot 2017-10-19 at 3 57 24 pm

SectionTable
screen shot 2017-10-19 at 3 53 09 pm

### CODE REVIEW

**Comment by caleybrock** [comment]
cc werketing @breville @tanyaparker @Erin007

**Comment by caleybrock** [comment]
Switching Brad out for Brent as reviewer since Brad has a presentation tomorrow.

**Comment by Erin007** [comment]
What happens when any of these tables are RTL?

File: apps/src/templates/tables/tableConstants.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import color from "../../util/color";
+import styleConstants from "../../styleConstants";
+
+// Constants for React tables
```
Comment (Bjvanminnen) [inline apps/src/templates/tables/tableConstants.js:4]: It seems like this entire file is generally stuff for reacttabular tables. I wonder if we should rename it to reflect that, i.e. `reactabularStyles.js` or `reactabularConstants.js`?

**Comment by caleybrock** [comment]
@Erin007 Good question. The styles on these are all the same as before, so RTL is the same as it currently looks. That being said I imagine there are some improvements we could make there, because it doesn't look like any of them have specific support.

File: apps/src/templates/tables/tableConstants.js (hunk 1 above)
Comment (caleybrock) [reply apps/src/templates/tables/tableConstants.js:4]: Although I didn't do this, I think html tables could be setup to work with these styles, so I purposefully kept it general.

**Pull request merged**